src/System.Configuration.ConfigurationManager/ref/System.Configuration.cs

```diff
@@ -1435,6 +1435,9 @@ public partial class DelegatingConfigHost : System.Configuration.Internal.IInter
         public virtual void VerifyDefinitionAllowed(string configPath, System.Configuration.ConfigurationAllowDefinition allowDefinition, System.Configuration.ConfigurationAllowExeDefinition allowExeDefinition, System.Configuration.Internal.IConfigErrorInfo errorInfo) { }
         public virtual void WriteCompleted(string streamName, bool success, object writeContext) { }
         public virtual void WriteCompleted(string streamName, bool success, object writeContext, bool assertPermissions) { }
+#if netfx
```

Comment:
Then they would need to be implemented in core as they are not part of it now, that is why I added the if because they are part of Desktop, so ApiCompat was complaining saying that they where in the implementation but not in the contract.